lib/logstash/filters/aggregate.rb

```diff
@@ -384,10 +433,31 @@ def remove_expired_maps()
     min_timestamp = Time.now - @timeout
```

Comment:
Can you update comment `If @push_previous_map_as_event option is set, ` to add "@push_map_as_event_on_timeout"